OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1468,7 +1451,9 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = "UnitTests/UnitTests-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
Making this change on the deployment target and the one below for macOS, drops the support for previous OS.
I am not sure that this is what we want because this is not what is exposed in the pod spec file (we support 7.0).
My point is that we could leave the Xcode 9 warning exposing the fact that it does not support iOS prior to 8.0 and macOS prior to 10.9 because, maybe, some people still use the previous versions.

Reply:
few lines below:
MACOSX_DEPLOYMENT_TARGET = 10.9;